src/ODEmodel.jl

```diff
@@ -40,12 +40,36 @@ function vTOg(v::AbstractVector)
     return G1, G2
 end
 
+""" To find the initial vector. """
+function residue(p, g1, g2)
+    t = LinRange(0.0, 0.5 * size(g1, 1), size(g1, 1))
+    g00 = g1[1, 1] + g2[1, 1]
+    return predict(p, g00, t, g1[:, 1], g2[:, 1])[1]
+end
+
+function optimise_control(g1, g2)
+    smin = [1e-4, 1e-4, 1e-4, 1e-4, 1e-4, 1e-4, 1e-4, 1e-4, 0.25];
+    smax = [2.0, 2.0, 2.0, 2.0, 2.0, 2.0, 2.0, 2.0, 0.75];
+    f(x) = residue(x, g1, g2)
+    return DrugResponseModel.optimize_helper(f, smin, smax, 100000)
+end
+
+function newPredict(p, pControl, t::Union{Real, LinRange}, g1data = nothing, g2data = nothing)
+
+    vStart = startV(pControl)
+    # Note that vStart is always scaled so the starting cell number is 1.0
+
+    return predict(p, vStart, t, g1data, g2data)
+end
```

Comment:
Is this a correct way of handling it? I am getting very high cost values.

Reply:
Looks right. Is the data scaled in such a way that the cell number at t=0 is 1.0? If not this vector needs to be scaled.

Reply:
Oh.. mo, this is my mistake. I will fix it, thank you.